lib/src/data/course.dart

```diff
@@ -0,0 +1,16 @@
+import "utils.dart";
+
+/// A course
+class Course {
+  /// Course id
+  int id;
```

Comment:
So this is just a temporary class, but you can just use
```dart
extension type CourseID(String value) { }
```
because the profiles will just be using IDs (see below)